wren-ui/src/apollo/server/services/queryService.ts

```diff
@@ -28,8 +28,8 @@ export interface ColumnMetadata {
 export interface PreviewDataResponse {
   correlationId?: string;
   processTime?: string;
-  columns: ColumnMetadata[];
-  data: any[][];
+  columns?: ColumnMetadata[];
```

Comment:
I prefer not to share type definitions here, as preview and dry-run have distinct return formats, and they should be defined separately. Can we define their types individually?

Reply:
Sure